recipes/cunit/all/conanfile.py

```diff
@@ -62,15 +62,23 @@ def source(self):
             for f in glob.glob("*.c"):
                 os.chmod(f, 0o644)
 
+    @property
+    def _user_build_info(self):
+        # If using the experimental feature with different context for host and
+        # build, the 'user_info' attributes of the 'build_requires' packages
+        # will be located into the 'user_info_build' object. In other cases they
+        # will be located into the 'deps_user_info' object.
+        return getattr(self, "user_info_build", None) or self.deps_user_info
```

Comment:
```suggestion
    @property
    def _user_info_build(self):
        return getattr(self, "user_info_build", self.deps_user_info)
```

Reply:
it's the exactly same effect, just a bit different syntax, right?
also, it loses the important comment explaining the motivation behind this function

Reply:
We already have this syntax in many recipes, it's more consistent.

Reply:
The comment can be thrown away.
Using `_user_info_build` is the new standard, so needing to place the comment everywhere would become cumbersome.